DataConnectors/ForgeRock_CEF.json

```diff
@@ -0,0 +1,84 @@
+{
+  "id" : "ForgeRock_Common_Audit_CEF",
+  "title" : "ForgeRock Common Audit for Sentinel",
+  "publisher" : "ForgeRock Inc",
+  "descriptionMarkdown" : "The ForgeRock Identity Platform provides a single common auditing framework. Extract and aggregate log data across the entire platform with common audit (CAUD) event handlers and unique IDs so that it can be tracked holistically. Open and extensible, you can leverage audit logging and reporting capabilities for integration with Microsoft Azure Sentinel via this CAUD for CEF connector.",
+  "graphQueries" : [
+    {
+      "metricName" : "Total data received",
+      "legend" : "DATATYPE_NAME",
+      "baseQuery" : "CommonSecurityLog​ | where DeviceVendor in (\"ForgeRock Inc\")\n| where DeviceAction =~ \"FAILED\""
+    }
+  ],
+  "sampleQueries" : [
+    {
+      "description" : "Total failed login attempts to the ForgeRock Platform",
+      "query" : "CommonSecurityLog | where DeviceVendor in (\"ForgeRock Inc\")\n| extend user = split(split(AdditionalExtensions, \";\", 4)[0], \"=\", 1)[0] // extract userID from  4th field in json payload\n| where strlen(user) > 2 // if not null\n| where DeviceAction =~ \"FAILED\""
+    },
+    {
+      "description" : "Autocluster logins by IP addresses ",
+      "query" : "CommonSecurityLog | where DeviceVendor in (\"ForgeRock Inc\")\n| where DeviceAction =~ \"FAILED\"\n| project SourceIP \n| evaluate autocluster(0.6)"
+    }
+  ],
+  "dataTypes" : [
+    {
+      "name" : "CommonSecurityLog (DATATYPE_NAME)",
+      "lastDataReceivedQuery" : "CommonSecurityLog | where DeviceVendor in (\"ForgeRock Inc\")\n| where DeviceProduct == \"Trust Partner Network\"\n| summarize Time = max(TimeGenerated)\n| where isnotempty(Time)"
+    }
+  ],
+  "connectivityCriterias" : [
+    {
+      "type" : "IsConnectedQuery",
+      "value" : [
+        "\nCommonSecurityLog\n| where DeviceVendor == \"PROVIDER NAME\"\n| where DeviceProduct == \"APPLIANCE NAME\"\n\n            | summarize LastLogReceived = max(TimeGenerated)\n            | project IsConnected = LastLogReceived > ago(30d)"
+      ]
+    }
+  ],
+  "availability" : {
+    "status" : 1
+  },
+  "permissions" : {
+    "resourceProvider" : [
+      {
+        "provider" : "Microsoft.OperationalInsights/workspaces",
+        "permissionsDisplayText" : "read and write permissions are required.",
+        "providerDisplayName" : "Workspace",
+        "scope" : "Workspace",
+        "requiredPermissions" : {
+          "read" : true,
+          "write" : true,
+          "delete" : true
+        }
+      }
+    ]
+  },
+  "instructionSteps" : [
+    {
+      "title" : "Configuration for the ForgeRock Common Audit (CAUD) for Microsoft Azure Sentinel",
+      "description" : "Install and configure the ForgeRock Common Audit (CAUD) for Microsoft Azure Sentinel per the documentation at https://github.com/javaservlets/SentinelAuditEventHandler",
```

Comment:
I reviewed the documentation hosted on GitHub https://github.com/javaservlets/SentinelAuditEventHandler - please update all references of "Sentinel" to "Azure Sentinel" which is the brand name of the product in public documentation.

Reply:
done